File: ProjectTemplates/MonoDevelop/MonoDevelop.MonoGame/templates/Common/Importer1.cs
using System;
using System.Collections.Generic;
using System.Linq;
using Microsoft.Xna.Framework;
using Microsoft.Xna.Framework.Graphics;
using Microsoft.Xna.Framework.Content.Pipeline;
using Microsoft.Xna.Framework.Content.Pipeline.Graphics;

using TImport = System.String;

namespace ${Namespace}
{
	[ContentImporter(".txt", DisplayName = "Importer1", DefaultProcessor = "Processor1")]
	public class Importer1 : ContentImporter<TImport>
	{
		public override TImport Import(string filename, ContentImporterContext context)
		{
			return default(TImport);
		}
	}
}

File: ProjectTemplates/DotNetTemplate/MonoGame.Templates.VisualBasic/content/PipelineExtension/Importer1.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports Microsoft.Xna.Framework
Imports Microsoft.Xna.Framework.Graphics
Imports Microsoft.Xna.Framework.Content.Pipeline
Imports Microsoft.Xna.Framework.Content.Pipeline.Graphics

Imports TImport = System.String

Namespace BlankApp
    <ContentImporter(".txt", DisplayName:="Importer1", DefaultProcessor:="Processor1")>
    Public Class Importer1 : Inherits ContentImporter(Of TImport)
        Public Overrides Function Import(ByVal filename As String, ByVal context As ContentImporterContext) As TImport
            Return Nothing
        End Function
    End Class
End Namespace

